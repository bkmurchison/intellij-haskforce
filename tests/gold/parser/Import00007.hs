module Import00007 where

import Servant ((:<|>)((:<|>)))
