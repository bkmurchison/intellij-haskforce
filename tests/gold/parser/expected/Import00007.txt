Haskell File
  MODULEDECL
    PsiElement(HaskellTokenType.module)('module')
    QCONID
      CONID
        PsiElement(HaskellTokenType.conidRegexp)('Import00007')
    PsiElement(HaskellTokenType.where)('where')
  BODY
    IMPDECL
      PsiElement(HaskellTokenType.import)('import')
      QCONID
        CONID
          PsiElement(HaskellTokenType.conidRegexp)('Servant')
      PsiElement(HaskellTokenType.()('(')
      IMPORTT
        TYCON
          PsiElement(HaskellTokenType.()('(')
          CONSYM
            PsiElement(HaskellTokenType.Consym)(':<|>')
          PsiElement(HaskellTokenType.))(')')
        PsiElement(HaskellTokenType.()('(')
        CON
          PsiElement(HaskellTokenType.()('(')
          CONSYM
            PsiElement(HaskellTokenType.Consym)(':<|>')
          PsiElement(HaskellTokenType.))(')')
        PsiElement(HaskellTokenType.))(')')
      PsiElement(HaskellTokenType.))(')')